### ISSUE TITLE
[Core] Fix Accessibility issue with responsive buttons

### ISSUE THREAD

**Issue by brandonrq506** [brandonrq506/laboratory#1601]
Because we used `hidden` instead of `sr-only` that means the labels are not accessible to screen readers.
